gems/aws-sdk-core/lib/aws-sdk-core/shared_config.rb

```diff
@@ -134,27 +122,28 @@ def assume_role_web_identity_credentials_from_config(profile)
       if @config_enabled && @parsed_config
         entry = @parsed_config.fetch(p, {})
         if entry['web_identity_token_file'] &&
-          entry['role_arn']
+           entry['role_arn']
```

Comment:
Spacing

Reply:
Without this space I get a rubocop warning: "RuboCop: Align the operands of a condition in an `if` statement spanning multiple lines."